controllers/configurationpolicy_controller.go

```diff
@@ -1913,37 +1865,24 @@ func (r *ConfigurationPolicyReconciler) getMapping(
 	ext runtime.RawExtension,
 	policy *policyv1.ConfigurationPolicy,
 	index int,
-) (mapping *meta.RESTMapping, updateNeeded bool) {
+) (mapping *meta.RESTMapping, result *objectTmplEvalResult) {
```

Comment:
The pointer is nice in that it prevents the policy data from being copied in memory, however, it does mean the function could unintentionally modify the policy.

What do you think @yiraeChristineKim and @JustinKuli?

Reply:
If our controller is out of memory, I agree to use a pointer. Otherwise, mappingErrResult is only used 
line 1159, so I vote for removing Pointer 
 ```
mappingErrResult != nil {
				nsToResults[ns] = *mappingErrResult

				continue
			}
```

Reply:
Oh no, get mapping is used only here... let's go with pointer to save memory